generator/src/main/java/org/javabubble/generator/site/ReadmeHTML.java

```diff
@@ -103,7 +103,8 @@ private DomContent export() {
 				ul( //
 						li(a("people.yaml").withHref(PeopleYaml.LOCATION), text(" - raw data")), //
 						li(a("people.json").withHref(PeopleJson.LOCATION), text(" - raw data")), //
-						li(a("following.csv").withHref(FollowingCSV.LOCATION), text(" - for Mastodon import")) //
+						li(a("following.csv").withHref(FollowingCSV.LOCATION), text(" - for Mastodon import")), //,
```

Comment:
```suggestion
						li(a("following.csv").withHref(FollowingCSV.LOCATION), text(" - for Mastodon following import")), //,
```

Clarify use case of the existing file